activity_calendar/views.py

```diff
@@ -365,3 +365,30 @@ def form_invalid(self, form):
         else:
             messages.error(self.request, _("Some input was not valid. Please correct your data below"))
             return super(CreateSlotView, self).form_invalid(form)
+
+
+# #######################################
+# ######   Activity Editing View   ######
+# #######################################
+
+
+class EditActivityMomentView(LoginRequiredMixin, PermissionRequiredMixin, ActivityMixin, FormView):
+    form_class = ActivityMomentForm
+    template_name = "activity_calendar/activity_moment_form_page.html"
+    permission_required = ('activity_moment.can_edit',)
+
+    def get_form_kwargs(self):
+        kwargs = super(EditActivityMomentView, self).get_form_kwargs()
+        kwargs.update({
+            'instance': self.activity_moment,
+        })
+        return kwargs
+
+    def form_valid(self, form):
+        form.save()
+        message = _("You have successfully changed the settings for '{activity_name}'")
```

Comment:
I suggest we also include the activity's date/time here

Reply:
Done